inlang/source-code/paraglide/paraglide-js/README.md

```diff
@@ -79,23 +86,62 @@ m.loginHeader({ name: "Samuel" }) // Hello Samuel, please login to continue.
 // change the language
 setLanguageTag("de")
 
-m.loginHeader({ name: "Samuel" }) // Hallo Samuel, bitte melde dich an, um fortzufahren.  
+m.loginHeader({ name: "Samuel" }) // Hallo Samuel, bitte melde dich an, um fortzufahren.
 ```
-Paraglide JS exports four variables and functions via "@inlang/paraglide-js":
+
+Paraglide JS provides five exports in `./paraglide/runtime.js`:
 
 | Variable | Description |
 | --- | --- |
-| `sourceLanguageTag` | The source language tag of the project |
+| `sourceLanguageTag` | The default language tag of the project |
```

Comment:
why default? it's the source language tag

Reply:
I changed it back